miio/airhumidifier.py

```diff
@@ -185,15 +185,29 @@ def motor_speed(self) -> Optional[int]:
 
     @property
     def depth(self) -> Optional[int]:
-        """The remaining amount of water in percent."""
+        """Return raw value of depth."""
```

Comment:
So, this is a breaking change but I don't think it's a big deal in this case.

One question though, should this property be marked as deprecated/should there be a logger warning guiding to use `water_level` instead of this?

Reply:
Good question. HA integration will use only `water_level` value. I think we should marked `depth` value as deprecated and remove it in, I don't know, maybe 3 months. What do you think?

Tomorrow I'll add a logger warning about the deprecation of `depth`.

Reply:
Sounds good to me! Those (if any) who want to access the raw value can always simply access it through the raw data. Let's merge this after adding a warning that the behavior has been changed and that using the new `water_level` and `water_tank_detached` is recommended.

Reply:
Done